can-control_test.js

```diff
@@ -489,28 +489,30 @@ test("Creating an instance of a named control without passing an element", funct
 
 	var MyControl = Control.extend('MyControl');
 	try {
-		new MyControl();	
+		new MyControl();
 	}
 	catch(e) {
 		ok(true, 'Caught an exception');
 	}
-	
+
 });
 
 test("Creating an instance of a named control passing a selector", function() {
 
 	this.fixture.appendChild( fragment('<div id=\'my-control\'>d</div>') );
-	
+
 	var MyControl = Control.extend('MyControl');
 	var myControlInstance = new MyControl('#my-control');
-	
+
 	ok(className.has.call(myControlInstance.element, 'MyControl'), "Element has the correct class name");
 });
 
 test('destroy should not throw when domData is removed (#57)', function () {
 	var Things = Control.extend({
 		destroy: function(){
-			domData.delete.call(this.element);
+      if (this.element) {
```

Comment:
@andrejewski not sure why this changed the spacing. Make sure you have editorconfig set up.